bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -1469,8 +1469,11 @@ private void checkStatus() {
             var commit = localRepo.lookup(amendedHash).orElseThrow();
             var commitMessage = String.join("\n", commit.message());
 
+            var onlyReviewersCheckFailed = visitor.errorFailedChecksMessages().stream()
+                    .allMatch(message -> message.contains("Too few reviewers"));
```

Comment:
This looks awkward. Can we extend the Visitor with an explicit method for this instead? The role of the Visitor implementation is to gather the jcheck errors and present them in a way that makes sense to the caller, in this case the pr bot classes.